assets/sass/_base.sass

```diff
@@ -205,12 +205,21 @@ pre
     ul
         margin-bottom: 1.25rem
         margin-left: 1.5em
-        list-style: none
+        list-style: disc
+    ul ul
+        margin-bottom: 0
     li
-        list-style-type: disc
         margin-left: 1.5em
         list-style-position: outside
         padding-left: 1em
+    ol
+        margin-bottom: 1.25rem
+        margin-left: 1.5em
+    ol ol
+        list-style-type: lower-alpha
+        margin-bottom: 0
+    ol ol ol
+        list-style-type: lower-roman
```

Comment:
Thanks for the fix :+1: 

Can you add an example similar to the one in the PR description to `exampleSite`

Reply:
Done 👍